nix/vast/default.nix

```diff
@@ -108,15 +82,29 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = lib.optional isStatic "pic";
 
-  doCheck = false;
+  doCheck = true;
+  # Concurrent testing is not possible because multiple suites reuse the same
+  # file paths.
```

Comment:
Which suites do this?

Reply:
I don't know exactly, I just noticed that I get lots of breakage when the tests are run in parallel. We normally never do that.